cirq/study/sweeps.py

```diff
@@ -126,7 +126,7 @@ def param_tuples(self) -> Iterator[Params]:
         yield ()
 
     def __repr__(self):
-        return 'Unit'
+        return 'cirq.Unit'
```

Comment:
Hm, this one was wrong before. Should be `cirq.UnitSweep`

Reply:
Done.